src/app/pages/Sandbox/Editor/Content/Header/ShareView.js

```diff
@@ -296,11 +343,25 @@ class ShareView extends React.PureComponent {
                       setValue={this.setHideNavigation}
                     />
                     <PaddedPreference
+                      title="Single file entry script"
```

Comment:
and honestly, i'd just put the moduleId in the URL. And here's why: if u choose the direction i've laid out, you're now trying to get the user to see that they can control what module is displayed--that's a core feature. So u want them to see it in the URL now. And here's what makes it more obvious: the id is the file name, not the real id. That way they can manually change it on their own, etc. It's the full url-encoded file path. 

So we should transition to file paths later (should u approve), and then now just always keep the module Id in the URL as that would be the direction we're heading. 

I can do this for u in the stage 2.